tests/test_crawl.py

```diff
@@ -305,10 +305,10 @@ def test_referer_header(self):
         # basic asserts in case of weird communication errors
         assert "responses" in crawler.spider.meta
         assert "failures" not in crawler.spider.meta
-        # start requests doesn't set Referer header
+        # test_start doesn't set Referer header
```

Comment:
I think there are some search+replace errors in several files that produced "test_start" or e.g. "process_test_start"